rcl/src/rcl/publisher.c

```diff
@@ -17,13 +17,13 @@ extern "C"
 {
 #endif
 
-#include "rcl/publisher.h"
```

Comment:
the style of this repo has the "matching" include for the file at the top so it seems this change should be reverted

Reply:
Moved back up.